pom.xml

```diff
@@ -1198,7 +1216,7 @@
   <!-- All non-build related project data below here -->
   <organization>
     <name>Opencast Project</name>
-    <url>http://www.opencastproject.org/</url>
+    <url>http://www.opencast.org/</url>
```

Comment:
originally I wanted to update this to https as well, but it's not implemented

Reply:
I know :(